packages/framework/src/Resources/views/Admin/Layout/layout.html.twig

```diff
@@ -28,6 +28,20 @@
                             <img class="web__header__logo__svg" src="{{ asset('public/admin/images/logo.svg') }}" alt="Shopsys Platform">
                         </a>
                     </div>
+                    <div class="web__header__search js-header-search">
```

Comment:
I'm thinking whether it would not be better to put the search input above the menu instead of in the header.

Like this, so the purpose is clearer.
![Snímek obrazovky 2024-12-18 v 16 51 02](https://github.com/user-attachments/assets/bb8d1209-4ab3-42d7-93ab-976d0003430f)